ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -394,9 +406,13 @@ Loader {
 
             readonly property int contentType: convertContentType(root.messageContentType)
             readonly property bool isReply: root.responseTo !== ""
-            readonly property var replyMessage: root.messageStore && isReply ? root.messageStore.getMessageByIdAsJson(root.responseTo) : null
+            property var replyMessage: getReplyMessage()
```

Comment:
@borismelnik this breaks a dependency... `root.responseTo` -> role `responseToMessageWithId`, so if you revert to line `397` you may not need connection in `ChatMessagesView.qml` line `245`.

Reply:
I thought about it, but what dependency do? we can't change `responseToMessageWIthId` and rising a signal `dataChange` with the same `responseToMessageWIthId` never change anything here :)

Reply:
@borismelnik didn't check the code, but I guess when some change for a message with id `0x1111` is received, that we're checking a model and if we find a message with id `0x2222` which is actually a reply to the message `0x1111` then we're emitting a dataChanged signal for message `0x2222` with role `responseToMessageWIthId`

Reply:
yes and `onDataChanged` should process it :)

Reply:
What I want to say... `dataChanged` signal is sent from multiple places with different role, in this particular case we're interesting to handle only the case when message with the id which is set as `responseToMessageWIthId` of a certain message/s is changed. If we have general slot, without any condition, `updateReplyInfo` will be called even for messages which are not a reply messages, for all messages in the model, but for reply messages it will be called twice, once because of line `442` once because of the line `421` of the `message_model.nim`

What would be the correct way (I thought it was like that before)... if any change for message B is received, than we're checking the model and search for a message/s which have id of message B set for `responseToMessageWIthId`, if there is/are such message/s we're emitting `dataChanged` signal with `responseToMessageWIthId` role for those messages, that triggers necessary changes on the qml side only in case it's really needed.